ci-operator/step-registry/cucushift/installer/rehearse/aws/cases/valid-lb-subnet/cucushift-installer-rehearse-aws-cases-valid-lb-subnet-workflow.yaml

```diff
@@ -0,0 +1,7 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-cases-valid-lb-subnet
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-aws-cases-valid-lb-subnet-provision
```

Comment:
The chain will create vpc on aws, we need a deprovision chain to destroy them, right ?

Reply:
I put it in `post_actions` function, and it will be invoked in trap.

Reply:
my bad, I missed that.